tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -47,7 +49,12 @@ preferably, RStudio loaded on your computer.
 
 ### Data to Download
 
-{% include/dataSubsets/_data_Imaging-Spec-Data-H5.html %}
+{% include/dataSubsets/_data_Imaging-Spec-Data-H5-2020.html %}
```

Comment:
Question just to confirm: You downloaded this data recently prior to adding to FigShare. If yes, ignore.  If no, we may need to check a few things.

Reply:
Yes, these data are based on the latest hyperspectral mosaic reflectance data available on th edata portal.